docs/reference/operators.rst

```diff
@@ -67,6 +67,12 @@ See the :ref:`Visibility System <visibility>` reference for a discussion of the
 ----------------------------------
 Whether a position or `Object` lies in the `Region`; for the latter, the object must be completely contained in the region.
 
+.. _({Object} | {region}) intersects ({Object} | {region}):
+
+(*Object* | *region*) intersects (*Object* | *region*)
+------------------------------------------------------
+Whether an `Object`/`Region` intersects another `Object`/`Region`, i.e. whether any portion of the occupied spaces intersect. It can often be useful to check intersection with the :term:`footprint` of a region, e.g. when checking whether a car intersects a given lane. In this case, one would write :scenic:`car intersects lane.footprint` instead of :scenic:`car intersects lane`. For more details, see :term:`footprint`.
```

Comment:
Let's put the footprint discussion in its own paragraph, starting with something like "When working with 2D regions, it can be useful...". That way people who aren't using 2D regions don't have to read that paragraph.

Reply:
Good idea